src/vs/workbench/contrib/chat/browser/actions/chatQuickInputActions.ts

```diff
@@ -249,10 +253,13 @@ class InteractiveQuickPickSession extends Disposable {
 
 	async accept(query: string) {
 		await this._model.waitForInitialization();
+		const requests = this._model.getRequests();
+		const lastRequest = requests[requests.length - 1];
+		if (lastRequest?.message && lastRequest?.message === query) {
```

Comment:
The `lastRequest?.message` condition is redundant

Reply:
will add in debt week